proto/schema/v1/schema.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package schema.v1;
+
+service SchemaService {
+  rpc Register(RegisterRequest) returns (RegisterResponse);
+  rpc Fetch(FetchRequest) returns (FetchResponse);
+}
+
+message Schema {
+  enum Type {
+    TYPE_UNSPECIFIED = 0;
+    TYPE_AVRO = 1;
+  }
+
+  // The schema's globally unique ID
+  string id = 1;
+  // The schema's name (globally unique)
+  string name = 2;
+  int32 version = 3;
```

Comment:
The name can probably be repeated in other schemas, but the name+version combination should be unique.